test/hotspot/jtreg/serviceability/dcmd/compiler/CompilerMemoryStatisticTest.java

```diff
@@ -39,18 +39,6 @@
  * @run main/othervm -XX:CompileCommand=memstat,*.* CompilerMemoryStatisticTest
  */
 
-/*
- * @test CompilerMemoryStatisticTest
- * @summary Test Compiler.memory
- * @requires vm.compiler1.enabled
- * @requires vm.compiler2.enabled
- *
- * @library /test/lib
- * @modules java.base/jdk.internal.misc
- *          java.management
- * @run main/othervm -XX:CompileCommand=memstat,*.*,collect CompilerMemoryStatisticTest
- */
-
```

Comment:
Reviewer hint: Removed because its redundant with the syntax check introduced in MemStatTest.java